### PR TITLE
chore: release v1.5.0

### PR DESCRIPTION
**Pull request by tao12345666333**
* bump version
* add changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -33,6 +34,302 @@
...
+Please try out the release binaries and report any issues at
+https://github.com/apache/apisix-ingress-controller/issues.
+
+### Contributors
+
+* Jintao Zhang
+* Sarasa Kisaragi
+* Xin Rong
+* John Chever
+* Xin Rong
```
Comment (bzp2010) [inline CHANGELOG.md:60]: repeated

File: CHANGELOG.md [hunk 2]
```diff
@@ -33,6 +34,302 @@
...
+* dependabot[bot]
+* nevercase
+* Gallardot
+* Nic
+* lsy
+* mango
+* Fatpa
+* Hoshea Jiang
+* JasonZhu
+* Xin Rong
```
Comment (bzp2010) [inline CHANGELOG.md:71]: repeated

Comment (juzhiyuan) [inline CHANGELOG.md:71]:
```suggestion
```

**Review by juzhiyuan — APPROVED** [review]

**Comment by juzhiyuan** [comment]
Hi, please have a look at the failed CI 🙂

**Pull request merged**